sections/calibration_products.tex

```diff
@@ -0,0 +1,659 @@
+% This section is subsubsection crazy and all the whitespace ends up looking absurd.
+% If needed, a similar block with the defaults can just be placed at the end of this section though.
+% And if people prefer it like this, it can be moved to the preamble.
+\titlespacing*{\subsubsection}
+{0pt}{1ex}{0ex}
+
+%%%%%%%%%% progress marker %%%%%%%%%%
+
+%\section{Calibration Products Pipeline\\(\wbsCPP)}
+\section{Calibration Products Production}
+\label{sec:cpp}
+
+This section details the input data and algorithms required to generate all data products necessary for the photometric calibration of the LSST survey. Details of the application of these products is covered in other sections of this document. The details of the input datasets are given in \secsymbol\ref{sec:CPP:inputs} and  \secsymbol\ref{sec:CPP:auxTelescope:inputs} , which define the source of these data, \ie which will be provided by the camera team and which will be measured on the mountain. Finally, sections \secsymbol\ref{sec:CPP:output} and \secsymbol\ref{sec:CPP:auxTelescope:outputs} list the various output data products from the Calibration Products Pipeline.
+
+\subsection{Key Requirements}
+\label{sec:CPP:keyRequirements}
+The work performed in this WBS serves several complementary roles:
+
+\begin{itemize}
+ \item It will enable the production of calibration data products as required by the Level 2 Photometric Calibration Plan (\NewPCP{}) and other planning documents \cite{Lupton15}. This includes both characterization of the sensitivity of the LSST system (optics, filters and detector) and the transmissivity of, and emission from, the atmosphere;
```

Comment:
We should not be citing Lupton15. We should decide whether this document should go on DocuShare or whether this document is no longer needed.